extensions/ONNX/Onnx/OnnxConfig.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#pragma warning disable IDE0130 // reduce number of "using" statements
+// ReSharper disable once CheckNamespace - reduce number of "using" statements
+namespace Microsoft.KernelMemory;
+public class OnnxConfig
+{
+    /// <summary>
+    /// Path to the directory containing the .ONNX file.
+    /// </summary>
+    public string ModelPath { get; set; } = string.Empty;
+
+    /// <summary>
+    /// The maximum length of the response that the model will generate. See https://onnxruntime.ai/docs/genai/reference/config.html
+    /// </summary>
+    public uint MaxLength { get; set; } = 2048;
+
+    /// <summary>
+    /// The minimum length of the response that the model will generate. See https://onnxruntime.ai/docs/genai/reference/config.html
+    /// </summary>
+    public uint MinLength { get; set; } = 0;
+
+    /// <summary>
+    /// The option to use a share one buffer for past and present for efficiency. See https://onnxruntime.ai/docs/genai/reference/config.html
```

Comment:
I checked the docs but I could not understand what this means. If the buffer is shared across two requests, is there a risk that two different users will interfere with each other, risking also to leak personal data?

Reply:
Will reach out to the OnnxRuntimeGenAi team about this one, but as mitigation to the security concern you have, I'll be removing this modeled concept until I get more clarity.